test/e2e/util_test.go

```diff
@@ -600,3 +600,28 @@ func buildCRCleanupFunc(c operatorclient.ClientInterface, apiGroup, version, nam
 		})
 	}
 }
+
+// Local determines whether test is running locally or in a container on openshift-CI.
+// Queries for a clusterversion object specific to OpenShift.
+func Local(client operatorclient.ClientInterface) (bool, error) {
```

Comment:
Did you consider separating the logic that copies images and provides the registry address into two implementations using build tags?

Reply:
That makes sense now that the two paths diverge pretty heavily, but originally the test looked the same for both. We could do that now but I'm not sure the additional effort is warranted given its passing as-is, the code is still pretty readable, and its better to focus on other tasks now.

Reply:
That's fine. Maybe it's something to consider the next time we introduce a test that has different local and non-local modes.